File: system-test/dotnet/SystemTests/Legacy/Mod08/SmokeTests/External/ErpSmokeTest.cs
using Common;
using SystemTests.Legacy.Mod07.Base;
using Shouldly;
using Xunit;

namespace SystemTests.Legacy.Mod08.SmokeTests.External;

public class ErpSmokeTest : BaseSystemDslTest
{
    [Fact]
    public async Task ShouldBeAbleToGoToErp()
    {
        (await _app.Erp().GoToErp()
            .Execute())
            .ShouldSucceed();
    }
}

File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/legacy/mod07/smoke/external/ErpSmokeTest.java
package com.mycompany.myshop.systemtest.legacy.mod07.smoke.external;

import com.mycompany.myshop.systemtest.legacy.mod07.base.BaseUseCaseDslTest;
import org.junit.jupiter.api.Test;

class ErpSmokeTest extends BaseUseCaseDslTest {
    @Test
    void shouldBeAbleToGoToErp() {
        app.erp().goToErp()
                .execute()
                .shouldSucceed();
    }
}


